app/src/main/res/values/strings.xml

```diff
@@ -197,17 +197,17 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="horizontal_space_between_textactions">Horizontal space between textaction buttons</string>
 
     <string name="center_text">Center text</string>
-    <string name="edit_text_in_center_of_screen">Edit text in center of the screen</string>
+    <string name="edit_text_in_center_of_screen">Edit text in the center of the screen</string>
     <string name="clipboard">Clipboard</string>
     <string name="select_existing_document">Select existing document</string>
     <string name="open_source_licenses">Open Source Licenses</string>
     <string name="rate_this_app">Rate this app</string>
     <string name="translate">Translate</string>
     <string name="more_info_and_help">More info and help</string>
     <string name="report_bugs">Report bugs</string>
-    <string name="more_info_on_how_to_donate_or_otherwise_support_this_project_text">More info on donating or supporting this project</string>
+    <string name="more_info_on_how_to_donate_or_otherwise_support_this_project_text">More info on donating to the developer, or contributing this project</string>
```

Comment:
Please just leave as is, don't want to specify here if it is 'the developer', 'the team', designer+developer ... or something else :-)

Reply:
Why?